lib/function/algebra/derivative.js

```diff
@@ -567,33 +567,42 @@ function factory (type, config, load, typed) {
       var arg2 = node.args[1];
 
       switch (node.op) {
-        case '+':
         case '-':
-          // d/dx(+/-f(x)) = +/-f'(x)
-          if (node.args.length == 1) {
-            return new OperatorNode(node.op, node.fn, [_derivative(arg1, constNodes)]);
-          }
-
+          if (node.args.length > 1)
```

Comment:
This should be `node.args.length > 2` (subtraction can be unary or binary).

Reply:
I think you're right joel. It would be great if we would have a unit test that would fail on this case. @HarrySarson would it be easy to cover this case with a unit test?